docs/getting-started/windows-standalone.md

```diff
@@ -0,0 +1,51 @@
+# Windows Standalone
+
+A Windows MSI package (64-bit only) is available to download at
+[https://github.com/signalfx/splunk-otel-collector/releases
+](https://github.com/signalfx/splunk-otel-collector/releases).
+
+## Getting Started
+
+### GUI
+
+Double-click on the downloaded package and follow the wizard.
+
+### PowerShell
+
+in a PowerShell terminal:
+
+```sh
+PS> Start-Process -Wait msiexec "/i PATH_TO_MSI /qn"
+```
+
+Replace `PATH_TO_MSI` with the *full* path to the downloaded package, e.g.
+`C:\your\download\folder\splunk-otel-collector-0.4.0-amd64.msi`.
```

Comment:
All instructions *after* this line are applicable for both GUI and command line installs. Maybe add a new "Post-Install" section?